src/CommunityToolkit.Maui.Core/Handlers/DrawingView/DrawingViewHandler.android.cs

```diff
@@ -0,0 +1,116 @@
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapClearOnFinish(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.SetClearOnFinish(view.ClearOnFinish);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.LineColor"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapLineColor(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.SetLineColor(view.LineColor);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.LineWidth"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapLineWidth(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.SetLineWidth(view.LineWidth);
+	}
+
+	/// <summary>
+	/// Action that's triggered when the DrawingView <see cref="IDrawingView.MultiLineMode"/> property changes.
+	/// </summary>
+	/// <param name="handler">An instance of <see cref="DrawingViewHandler"/>.</param>
+	/// <param name="view">An instance of <see cref="IDrawingView"/>.</param>
+	public static void MapMultiLineMode(DrawingViewHandler handler, IDrawingView view)
+	{
+		handler.PlatformView.SetMultiLineMode(view.MultiLineMode);
+	}
+	/// <inheritdoc />
+	protected override void ConnectHandler(MauiDrawingView nativeView)
+	{
+		base.ConnectHandler(nativeView);
+		nativeView.Initialize();
+		VirtualView.Lines.CollectionChanged += OnLinesCollectionChanged;
+		PlatformView.DrawingLineCompleted += OnPlatformViewDrawingLineCompleted;
+	}
+
+	/// <inheritdoc />
+	protected override void DisconnectHandler(MauiDrawingView nativeView)
+	{
+		PlatformView.DrawingLineCompleted -= OnPlatformViewDrawingLineCompleted;
+		VirtualView.Lines.CollectionChanged -= OnLinesCollectionChanged;
+		nativeView.CleanUp();
+		base.DisconnectHandler(nativeView);
+	}
+
+	/// <inheritdoc />
+	protected override MauiDrawingView CreatePlatformView() => new(Context);
+
+	void OnPlatformViewDrawingLineCompleted(object? sender, MauiDrawingLineCompletedEventArgs e)
+	{
+		VirtualView.DrawingLineCompleted(new DrawingLineCompletedEventArgs()
+		{
+			LineColor = e.Line.LineColor,
+			EnableSmoothedPath = e.Line.EnableSmoothedPath,
+			Granularity = e.Line.Granularity,
+			LineWidth = e.Line.LineWidth,
+			Points = e.Line.Points
+		});
+	}
+
+	void OnLinesCollectionChanged(object? sender, System.Collections.Specialized.NotifyCollectionChangedEventArgs e)
+	{
+		UpdateLines(VirtualView, PlatformView);
+	}
+
+	static void UpdateLines(IDrawingView virtualView, MauiDrawingView platformView)
```

Comment:
This should be an extension method

Reply:
But it's already an extension method. Am I missed something?

Reply:
@VladislavAntonyuk this is the one that should be an extension method... I looked on my mobile device and messed up with the ideas, sorry about that